### ISSUE TITLE
arm32, x86,x64 support

### ISSUE THREAD

**Issue by steelbytes** [Insta360Develop/CameraSDK-Android#83]
I need arm32, x86,x64 support.  Uselsess without.

**Comment by steelbytes** [comment]
so is this a dead project?  no resonses?

**Comment by steelbytes** [comment]
yeah it's dead.  maybe I'll have to reverse engineer their app :-(

**Comment by Antiglobalist** [comment]
It is not dead. 
But it seems like they don't care about Github Repo and dev feedbacks.

We work with Insta team directly and maybe will ask them to add more arch's

**Comment by Double-Insta360** [comment]
If you require an ARM version, please contact us at Enterprise@insta360.com.

**Comment by steelbytes** [comment]
For my app Insta360 is an optional feature so I don't mind having this feature only work on ARM64.

AND I do need it to work in the emulator.

So in `app\src\main\jniLibs` I made folders `armeabi-v7a`, `x86`, `x86_64` and added 0 byte dummy files for `libc++_shared.so`, `libOne.so`, `libusb-1.0.so` etc.  Then in my code I can check `Build.SUPPORTED_ABIS.contains("arm64-v8a")` before running any Insta360 code
